sympy/stats/tests/test_symbolic_probability.py

```diff
@@ -66,7 +66,6 @@ def test_literal_probability():
     assert Covariance(X, X**2).doit() == Covariance(X, X**2)
     assert Covariance(X, sin(X)).doit() == Covariance(sin(X), X)
     assert Covariance(X**2, sin(X)*Y).doit() == Covariance(sin(X)*Y, X**2)
-    assert Covariance(w, X).evaluate_integral() == 0
```

Comment:
Were the deleted results wrong?

Reply:
Oops, will add them. Actually while debugging an error, I deleted it mistakenly